velox/common/memory/MallocAllocator.h

```diff
@@ -116,12 +116,12 @@ class MallocAllocator : public MemoryAllocator {
 
   void* allocateZeroFilledWithoutRetry(uint64_t bytes) override;
 
-  /// Increment current usage and check current allocator consistency to make
```

Comment:
These should be `///` as they are multi-line comments?
https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md